commands/watch.ts

```diff
@@ -1,32 +1,53 @@
-// @ts-nocheck
-const fs = require('fs');
-const path = require('path');
-
-const { watch } = require('@hubspot/local-dev-lib/cms/watch');
-const { getCwd } = require('@hubspot/local-dev-lib/path');
-const { logger } = require('@hubspot/local-dev-lib/logger');
-
-const {
-  addConfigOptions,
-  addAccountOptions,
-  addCmsPublishModeOptions,
-  addUseEnvironmentOptions,
-  addGlobalOptions,
-  getCmsPublishMode,
-} = require('../lib/commonOpts');
-const { uploadPrompt } = require('../lib/prompts/uploadPrompt');
-const { validateCmsPublishMode } = require('../lib/validation');
-const { trackCommandUsage } = require('../lib/usageTracking');
-const { i18n } = require('../lib/lang');
-const { getUploadableFileList } = require('../lib/upload');
-const { logError, ApiErrorContext } = require('../lib/errorHandlers/index');
-
-const { EXIT_CODES } = require('../lib/enums/exitCodes');
-
-exports.command = 'watch [src] [dest]';
-exports.describe = i18n(`commands.watch.describe`);
-
-exports.handler = async options => {
+import fs from 'fs';
+import path from 'path';
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { AxiosError } from 'axios';
+
+import { watch } from '@hubspot/local-dev-lib/cms/watch';
+import { getCwd } from '@hubspot/local-dev-lib/path';
+import { logger } from '@hubspot/local-dev-lib/logger';
+
+import { getCmsPublishMode } from '../lib/commonOpts';
+import { uploadPrompt } from '../lib/prompts/uploadPrompt';
+import { validateCmsPublishMode } from '../lib/validation';
+import { trackCommandUsage } from '../lib/usageTracking';
+import { i18n } from '../lib/lang';
+import { getUploadableFileList } from '../lib/upload';
+import { logError, ApiErrorContext } from '../lib/errorHandlers';
+import { EXIT_CODES } from '../lib/enums/exitCodes';
+import { makeYargsBuilder } from '../lib/yargsUtils';
+import {
+  AccountArgs,
+  CmsPublishModeArgs,
+  CommonArgs,
+  ConfigArgs,
+  EnvironmentArgs,
+  YargsCommandModuleBucket,
+} from '../types/Yargs';
+import { WatchErrorHandler } from '@hubspot/local-dev-lib/types/Files';
+
+type WatchCommandArgs = ConfigArgs &
+  AccountArgs &
+  EnvironmentArgs &
+  CommonArgs &
+  CmsPublishModeArgs & {
+    src?: string;
+    dest?: string;
+    fieldOptions?: string[];
+    remove?: boolean;
+    initialUpload?: boolean;
+    disableInitial?: boolean;
+    notify?: string;
+    convertFields?: boolean;
+    saveOutput?: boolean;
+  };
+
+const command = 'watch [src] [dest]';
+const describe = i18n(`commands.watch.describe`);
+
+const handler = async (
+  options: ArgumentsCamelCase<WatchCommandArgs>
```

Comment:
nit, but for consistency we're using `args` in the handler now

Reply:
Oh dang, I've been using options everywhere